contracts/PetPark.sol

```diff
@@ -3,5 +3,89 @@ pragma solidity ^0.8.0;
 
 
 contract PetPark {
+    enum AnimalType {None, Fish, Cat, Dog, Rabbit, Parrot}
+    enum Gender {Male, Female}
+    address owner;
 
+    event Added(AnimalType, uint);
+    event Borrowed(AnimalType);
+
+    struct ClientProfile {
+        uint age;
+        Gender gender;
+        address wallet;
```

Comment:
You don't need to store this address here since it is already available in mapping where you index this by address